src/rviz/visualization_frame.h

```diff
@@ -343,7 +353,7 @@ protected Q_SLOTS:
   };
   QList<PanelRecord> custom_panels_;
 
-  QAction* add_tool_action_;
+  QAction* toolbar_button_separator_;
```

Comment:
This header is installed, so changing this variable name will break ABI. Mind keeping the old name and add a note that the variable should be renamed on Noetic?